flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/reader/deserializer/KafkaRecordDeserializationSchemaTest.java

```diff
@@ -93,6 +112,29 @@ public void testKafkaValueDeserializerWrapper() throws Exception {
         assertEquals("world", collector.list.get(0));
     }
 
+    @Test
+    public void testNoConfigurableKafkaSerializer() throws Exception {
+        final Map<String, String> config = ImmutableMap.of("simpleKey", "simpleValue");
+        KafkaRecordDeserializationSchema<String> schema =
+                KafkaRecordDeserializationSchema.valueOnly(SimpleStringSerializer.class, config);
+        schema.open(new TestingDeserializationContext());
+        Assertions.assertEquals(configuration, config);
+        assertFalse(isKeyDeserializer);
+        assertTrue(configurableConfiguration.isEmpty());
+    }
+
+    @Test
+    public void testConfigurableKafkaValueSerializer() throws Exception {
```

Comment:
nits: The test name should probably mention `deserializer` instead of `serializer`.

How about naming it `testKafkaValueDeserializerWrapperWithConfigurable` so that its name is more consistent with the existing `testKafkaValueDeserializerWrapper`?